esrally/mechanic/mechanic.py

```diff
@@ -270,6 +269,12 @@ def receiveMsg_StartEngine(self, msg, sender):
 
         if msg.external:
             self.logger.info("Cluster will not be provisioned by Rally.")
+            if msg.cluster_settings:
+                import json
```

Comment:
It was more to avoid that usage of the `json` module does not spread across `mechanic` because I felt it should not. I changed it now to a top-level import.